pghoard/pghoard.py

```diff
@@ -796,6 +797,10 @@ def run(self):
                 self.log.error("Unexpected critical failure in PGHoard thread. Quitting main loop.")
                 self.quit()
 
+    def send_periodic_metrics(self) -> None:
+        running_compression_threads_pc = sum([int(compressor.running) for compressor in self.compressors]) / len(self.compressors) * 100
```

Comment:
I wonder if `is_alive()` would be safer than `running` as there could be some unexpected errors where the thread dies without setting running to false. (Or instead of that add try/finally in `CompressorThread` that ensures `running` really is always set to false, which might be cleaner overall.)